data/human/outfits.txt

```diff
@@ -290,6 +290,19 @@ outfit "Thermal Scanner"
 	"thermal scan power" 35
 	description "While thermal signatures have always been useful as a means of identifying and tracking ships, determining a ship's effective average thermal load was functionally useless until recently. With the development of the plasma cannon and then the flamethrower, attempting to overheat a ship has become a viable combat technique. As such, engineers with the Free World have begun assembling these sensor packages to support the daring captains who choose to use this tactic."
 
+
+outfit "Astronomical Scanner"
+	category "Systems"
+	cost 18000
+	thumbnail "outfit/astronomical scanner"
+	"mass" 2
+	"outfit space" -2
+	"energy consumption" .1
+	"solar scan" 1
+	"ramscoop scan" 1
+	description "	An astronomical scanner provides data on the system's total solar power and wind output, allowing captains to see which systems are best for solar collection or ramscoop usage." 
```

Comment:
```suggestion
	description "An astronomical scanner provides data on the system's total solar power and wind output, allowing captains to see which systems are best for solar collection or ramscoop usage." 
```

Reply:
Sorry, forgot this game refuses to indent.

Reply:
No problem.

Reply:
I specifically have [#5342](https://github.com/endless-sky/endless-sky/pull/5342/files) implemented so all my vanilla outfit descriptions are properly indented along with my ships because it's not the 1600s where paper was expensive and page space was at a premium. But I digress lol.